content/docs/user-guide/packaging/project-export-ios.md

```diff
@@ -0,0 +1,84 @@
+```
+<O3DE_ENGINE>\scripts\o3de register -pp C:\path\to\o3de-atom-sampleviewer
+```
+
+You must also have a valid copy of Xcode installed on your machine, and a valid Apple Developer ID associated with that IDE. To set up such an ID, go to [developer.apple.com](https://developer.apple.com).
+
+To use your desired Apple ID, in Xcode go to `Xcode -> Settings`. In the Settings window, go to the Accounts tab and click the '+' icon at the bottom of the left panel to add your account. Select Apple ID to proceed, and follow on-screen instructions.
+
+You will also need to ensure that Xcode has the necessary SDKs installed for macOS and iOS, and that your preferred iOS test device is compatible with Xcode.
+
+## Quickstart
+### Running the Export Script
+Assuming you are just following along using the O3DE Atom Sampleviewer project, this single invocation should be all you need to generate the necessary Xcode project file in the project's build folder:
+```
+<O3DE_ENGINE>\scripts\o3de export-project -es <O3DE_ENGINE>\scripts\o3de\ExportScripts\export_source_ios_xcode.py -pp C:\path\to\o3de-atom-sampleviewer
+```
+Assuming the underlying CMake build system finds no issues with your engine or project installation, you should have the corresponding Xcode project file, which will be located at `<PROJECT>\build\game_ios`. For the Atom SampleViewer, it should be called `AtomSampleViewer.xcodeproj`.
+
+### Using Xcode
+First double click the file `AtomSampleViewer.xcodeproj` to open it in Xcode. This should load everything needed to build the app on an iOS device.
+
+First, make sure that you have the provisioning profile setup correctly. This will handle all code-signing of the application when deploying to your iOS device (this applies even to test builds).
+
+First click on the AtomSampleViewer project icon on the Explorer tree-view panel on the left hand side of the Xcode IDE to show the project configuration details. Then on the left column under the general tab, make sure to select the build configuration `AtomSampleViewer.GameLauncher`. The resulting configuration page should look like this:
+
+{{< image-width "/images/user-guide/packaging/project-export-ios/atom-sampleviewer-configuration-page.png" "700">}}
+
+Then go to the signing and capabilities tab, and setup the provisioning profile to link to your AppleID, slightly customize the bundle identifier (add any string you want, for example: 'test'), and check "Automatically manage signing."
+
+{{< image-width "/images/user-guide/packaging/project-export-ios/signing-and-capabilities-page.png" "700">}}
+
+Now click on the bar in the region labeled “AtomSampleViewer.GameLauncher”, and scroll through the drop down to click “Edit Scheme.”
+
+{{< image-width "/images/user-guide/packaging/project-export-ios/edit-scheme.png" "700">}}
+
+Set the desired build configuration, and in the arguments section, set the relevant CLI parameters. An example configuration is shown below:
```

Comment:
it might be good to mention that normally you would specify the commands to run on launch for a release game in code or by using the `autoexec.client.setreg` like so https://docs.o3de.org/docs/user-guide/packaging/windows-release-builds/#set-the-starting-level These launch parameters specified in the schema likely will be removed the next time you configure with CMake